_delphi_utils_python/data_proc/geomap/geo_data_proc.py

```diff
@@ -363,6 +365,25 @@ def derive_zip_population_table():
     df = census_pop.merge(fz_df, on="fips", how="left")
     df["pop"] = df["pop"].multiply(df["weight"], axis=0)
     df = df.drop(columns=["fips", "weight"]).groupby("zip").sum().dropna().reset_index()
+    ## filling population NAs for specific zips on zip_pop_missing Issue #0648
+    ## cheking if each zip still missing, and concatenating if True
+
+    zip_pop_missing = pd.DataFrame(
+        {
+            "zip": ['57756', '57764', '57770', '57772', '57794', '99554', '99563', '99566',
+                    '99573', '99574', '99581', '99585', '99586', '99604', '99620', '99632',
+                    '99650', '99657', '99658', '99662', '99666', '99677', '99686', '99693'],
+            "pop": [1126, 1923, 5271, 2048, 644, 677, 938, 192,
+                    1115, 2348, 762, 417, 605, 1093, 577, 813,
+                    568, 329, 329, 480, 189, 88, 4005, 248]
+        }
```

Comment:
Could you stuff this into a csv file instead? It will be easier to maintain if it's not hard-coded.

Reply:
Done